pkg/cli/admin/release/mirror.go

```diff
@@ -213,6 +371,29 @@ func (o *MirrorOptions) Run() error {
 		return fmt.Errorf("--skip-release-image and --to-release-image may not both be specified")
 	}
 
+	if o.ApplyReleaseImageSignature {
+		if len(o.ReleaseImageSignatureToDir) != 0 {
+			return fmt.Errorf("--release-image-signature-to-dir and --apply-release-image-signature may not both be specified")
+		}
+	} else {
+		if len(o.ReleaseImageSignatureToDir) == 0 {
+			if len(o.ToDir) == 0 {
+				return fmt.Errorf("if --to-dir and --apply-release-image-signature are not specified, --release-image-signature-to-dir must be used to specify a directory to export the signature")
+			}
+			o.ReleaseImageSignatureToDir = o.ToDir + configFilesBaseDir
+		}
+		if o.Overwrite {
+			return fmt.Errorf("--overwite is only valid when --apply-release-image-signature is specified")
+		}
+	}
+	return nil
+}
+
+func (o *MirrorOptions) Run() error {
+
+	if err := o.validate(); err != nil {
```

Comment:
You don't call validate here, this should live between these two lines above:
```go
kcmdutil.CheckErr(o.Complete(cmd, f, args))
kcmdutil.CheckErr(o.Run())
```
compare https://github.com/openshift/oc/blob/master/pkg/cli/extract/extract.go#L83-L87, for example.

Reply:
All done

Reply:
Moved Validate per referenced pattern.